docs2/site/docs/migrations/migration5.md

```diff
@@ -82,6 +82,42 @@ a list or array of a single item. For example, `{"query":"{ hero }"}` deserializ
 serializing objects to and from `string` values. For the `GraphQL.SystemTextJson`
 and `GraphQL.NewtonsoftJson` libraries, these serialize and deserialize to JSON strings.
 
+### 6. `AutoRegisteringObjectGraphType` and `AutoRegisteringInputObjectGraphType` enhancements
+
+#### Overridable base functionality
+
+The classes can be overridden, providing the ability to customize behavior of automatically
+generated graph types. For instance, to exclude properties marked with a custom attribute
+called `[InternalUse]` you could write this:
+
+```csharp
+private class CustomAutoObjectType<T> : AutoRegisteringObjectGraphType<T>
+{
+    protected override IEnumerable<FieldType> CreateFieldTypeList()
+    {
+        var props = GetRegisteredProperties();
+        foreach (var prop in props)
+        {
+            if (prop.GetCustomAttribute<InternalUseAttribute>() != null)
+                yield return CreateFieldType(prop);
+        }
+    }
+}
+```
+
+Similarly, by overriding `CreateFieldType` you can change the default name, description,
+graph type, or other information applied to each generated field.
+
+If you utilize dependency injection within your schema, you can register your custom graph
+type over top of the usual one as follows:
```

Comment:
usual ?

Reply:
Umm, yes?

Reply:
usual type?

Reply:
usual aka normal aka standard.....

Reply:
Do you like "standard" better?  There's probably a better way to word it, for sure.  I was trying to say that you can register a custom generic auto-registering graph type class.... to be instantiated instead of the.... i'm stumbling over my own words again.  You can see from the demo what I was trying to say.  Feel free to suggest a better sentence here.

Reply:
Maybe this?

```suggestion
type to be used instead of the standard type as follows:
```

Reply:
standard -> built-in ? (if we are talking about autoregisteringgraphtype)

Reply:
```suggestion
type to be used instead of the built-in type as follows:
```